src/views/Feed/components/UserGroupTable/UserGroupTable.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { FC } from 'react';
+import {
+    Grid,
+    Box,
+    Table,
+    TableHead,
+    TableRow,
+    TableCell,
+    TableBody,
+    Theme,
+    Button
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+import { UserGroupState, UserState } from '../../../../store/user/types';
+import { TableRow as ITableRow } from '../../../../store/groups/types';
+import UserTableRow from '../../../GroupTable/components/UsersTable/UserTableRow'
+import getOrdinalPosition from '../../../../services/utils/getOrdinalPosition';
+import { PlainFunction } from '../../../../types/interfaces'
+
+const useStyles = makeStyles((theme: Theme) => ({
+	gridContainer: {
+		display: 'flex',
+    },
+    content: {
+		flexDirection: 'column',
+		boxSizing: 'border-box',
+		borderRadius: 10,
+		padding: 30,
+		backgroundColor: theme.palette.background.paper,
+		marginBottom: '5%',
+    },
+    header: {
+		backgroundColor: theme.palette.secondary.dark,
+	},
+    table: {
+		backgroundColor: theme.palette.background.paper,
+    },
+    ordinalEnding: {
+		fontSize: 10,
+	},
+	groupName: {
+		fontWeight: theme.typography.h1.fontWeight,
+		color: theme.palette.primary.contrastText,
+		fontSize: theme.typography.subtitle1.fontSize,
+		borderBottom: 'none',
+		cursor: 'pointer',
+	},
+	viewTable: {
+		marginTop: 20,
+    },
+    headerCell: {
+		borderBottom: 'none',
+		fontSize: theme.typography.subtitle2.fontSize,
+		color: theme.palette.primary.contrastText,
+	},
+}));
+
+interface Props {
+    groups: UserGroupState[],
+		user: UserState,
+		groupNameClicked: PlainFunction,
+		showTableClicked: PlainFunction,
+		userNameClicked: PlainFunction,
+}
+
+const UserGroupTable: FC<Props> = (props) => {
+    const { groups, user, groupNameClicked, showTableClicked, userNameClicked } = { ...props };
+    const classes = useStyles();
+
+    return (
+        <Grid container className={classes.gridContainer} spacing={2}>
+					{groups.map((group: UserGroupState) => (
+						<Grid
+							item
+							key={group.groupId}
+							xs={12}
+							sm={6}
+							lg={4}
+							xl={4}
+							data-cy='feed-group'
+						>
+						<Box className={classes.content} display="flex">
+							<Box boxShadow={3}>
+								<Table className={classes.table}>
```

Comment:
how about moving the table to a different component?
will be easier to read, and might be reusable elsewhere